<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QMainWindow" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Executable File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Applcation Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Application Notes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Icontonic</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="appNameTxt">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="appNoteTxt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="iconTxt">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>401</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="execTxt">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>401</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="execBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isTerminalChkBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>471</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Show terminal when running (useful when view log information)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isRootChkBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>451</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Run under root privilege (useful when access system resource)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="iconBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>581</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Additional parameters (use '=' to separates key-value and ';' to separates parameters</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="parametersTxt">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>571</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="okBtn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>350</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileNameTxt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>571</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Categories (use ';' to separates each category)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="categoriesTxt">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>571</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="quitBtn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>350</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previewBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Preview</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
